includes/class-wc-emails.php

```diff
@@ -395,7 +395,12 @@ private function get_blogname() {
 	 */
 	public function low_stock( $product ) {
 		$subject = sprintf( '[%s] %s', $this->get_blogname(), __( 'Product low in stock', 'woocommerce' ) );
-		$message = sprintf( __( '%s is low in stock.', 'woocommerce' ), html_entity_decode( strip_tags( $product->get_formatted_name() ), ENT_QUOTES, get_bloginfo( 'charset' ) ) ) . ' ' . sprintf( __( 'There are %d left', 'woocommerce' ), html_entity_decode( strip_tags( $product->get_total_stock() ) ) );
+		/* translators: 1: product name 2: items in stock */
+		$message = sprintf(
+			__( '%1$s is low in stock. There are %2$d left.', 'woocommerce' ),
+			html_entity_decode( strip_tags( $product->get_formatted_name() ), ENT_QUOTES, get_bloginfo( 'charset' ) ),
+			html_entity_decode( strip_tags( $product->get_total_stock() ) )
+		);
```

Comment:
Merging two strings.
OLD: `%s is low in stock.` + `There are %d left`
New: `%1$s is low in stock. There are %2$d left.`